src/com/javabasic/lesson14oop/HotDrink.java

```diff
@@ -0,0 +1,28 @@
+package com.javabasic.lesson14oop;
+
+import java.util.concurrent.TimeUnit;
+
+public class HotDrink {
```

Comment:
Create a separate implementation (class) for each drink, for example, Coffee, Tea, etc inherited from HotDrink - OOP principle Inheritance
The `makeDrink` method should be implemented differently in each class - OOP principle Polymorphism.
Price should be final, set with constructor without setter - OOP principle Encapsulation.

Reply:
Done